dashboard/app/controllers/level_sources_controller.rb

```diff
@@ -10,6 +10,7 @@ class LevelSourcesController < ApplicationController
   before_action :set_level_source
 
   def show
+    @level_source.try(:replace_old_when_run_blocks)
```

Comment:
Is this the right place for this? We could arguable leave this one alone, since any shareable levels should have already been converted. Thoughts?